### ISSUE TITLE
It seems to be pretty hard to find a mention you just got seconds or minutes ago locally on that same session

### ISSUE THREAD

**Issue by ell1e** [vector-im/element-web#23503]
### Steps to reproduce

1. Get a desktop notification about a message with a mention and it disappears/you click it away being busy
2. After you're no longer busy some seconds or minutes later, you get curious and open Element to see what it's about
3. You click the notifications bell button to find the message. That failing, you search the new message marker for the channel and click the channel name that is marked to find the message. In short, you just try to see the mention.

#### What did you expect?
The message always shows in the notifications unless I restarted the client since. Clicking the room entry marked for new notification always jumps to that mention (or another newer mention in that same room if any since) if I haven't been looking at that room since, not the latest message, and visually flashes it somehow in the log for a brief second.

(I know for E2ee, notifications can't be fetched from the server and/or synced between clients. But Element should IMHO do the next best thing, which is keep the mention notifications in memory, locally, of all the mentions it actually happened to process on that device recently, until it restarts.)

#### What happened instead?
95% of the time it appears to be impossible to locate the message with the mention. It's not in the notifications bell button list. It's not marked, shown, in anyway jumped to clicking the room with the mention maker which is just showing the newest regular one instead. If the room has any higher amount of activity at all, it's basically now a matter of minutes to hours to possibly ever find that message.

For something like a messenger, I feel like something as basic as mentions should really work, it's frustrating!

### Operating system

Linux

### Application version

Element version: 1.11.8 Olm version: 3.2.12

### How did you install the app?

flathub flatpak

### Homeserver

matrix.org

### Will you send logs?

No

**Comment by daniellekirkwood** [comment]
@weeman1337 & @gaelledel 
This is related to the Notifications panel & the discussion around Unreads as a tab. Worth keeping in mind as we continue to talk through these items.

**Comment by Merrit** [comment]
Has there been any change with this? I can see literally no sort of notifications center in the interface to even reference, I would expect there to be some sort of central notifications center in the GUI to show me everything that needs my attention.

![image](https://github.com/element-hq/element-web/assets/9575627/46b32a52-c57a-4548-be73-700760c3f888)

**Comment by hasufell** [comment]
Yes, it's atrocious user experience. I have to use the search feature to find the mention.

**Comment by t3chguy** [comment]
The notification panel in Labs is the way around this

<img width="668" alt="image" src="https://github.com/element-hq/element-web/assets/2403652/4ade4b30-ada0-47f6-aae1-3885f5337e5b">

**Comment by Merrit** [comment]
> The notification panel in Labs is the way around this
> <img alt="image" width="668" src="https://private-user-images.githubusercontent.com/2403652/335976879-4ade4b30-ada0-47f6-aae1-3885f5337e5b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTc0MjM1ODksIm5iZiI6MTcxNzQyMzI4OSwicGF0aCI6Ii8yNDAzNjUyLzMzNTk3Njg3OS00YWRlNGIzMC1hZGEwLTQ3ZjYtYWFlMS0zODg1ZjUzMzdlNWIucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDYwMyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA2MDNUMTQwMTI5WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZmE4OTc0MTdlNDg3YzI1NWMwYmQzZjhjZTY4MTAyY2YzM2E3YWQ3MGM5MzExNGYzZTFkZjJhMzQxODdjZDk4MCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.GdGTw3giGkSAF5F10DUF52HbslaH2L829N9EN9XCq0A">

Where is this setting? I've enabled the notifications beta:

![image](https://github.com/element-hq/element-web/assets/9575627/009add4c-3fff-4313-bcbc-ab911c9d3550)

I still don't see a setting for a notifications panel anywhere.

**Comment by t3chguy** [comment]
You need an instance with full labs enabled, like develop.element.io.

Settings > Labs > Enable the notifications panel in the room header

It is entirely unrelated to the notification settings beta.

**Comment by Merrit** [comment]
Thanks, I see it on `develop.element.io`.

It does help to find notifications, though it appears to be stuck with the unread dot even after looking at notifications.
